draft-ietf-quic-transport.md

```diff
@@ -2665,15 +2674,39 @@ for some applications.
 ## Stream Identifiers {#stream-id}
 
 Streams are identified by an unsigned 32-bit integer, referred to as the Stream
-ID.  To avoid Stream ID collision, clients MUST initiate streams using
-odd-numbered Stream IDs; servers MUST initiate streams using
-even-numbered Stream IDs. If an endpoint receives a frame which
-corresponds to a stream which is allocated to it (i.e., odd-numbered for
-the client or even-numbered for the server) but which it has not yet
-created, it MUST close the connection with error code STREAM_STATE_ERROR.
+ID.  The least significant two bits of the Stream ID are used to identify the
+type of stream (unidirectional or bidirectional) and the initiator of the
+stream.
+
+The least significant bit (0x1) of the Stream ID identifies the initiator of the
+stream.  Clients initiate even-numbered streams (those with the least
+significant bit set to 0); servers initiate even-numbered streams (with the bit
+set to 1).  Separation of the stream identifiers ensures that client and server
+are able to open streams without the latency imposed by negotiating for an
+identifier.
+
+If an endpoint receives a frame for a stream that it expects to initiate (i.e.,
+odd-numbered for the client or even-numbered for the server), but which it has
+not yet opened but which it has not yet created, it MUST close the connection
```

Comment:
Some duplicate text here 'but which it has not yet opened but which it has not yet created'